src/main/java/io/reactivex/FlowableEmitter.java

```diff
@@ -17,7 +17,7 @@
 import io.reactivex.functions.Cancellable;
 
 /**
- * Abstraction over a RxJava Subscriber that allows associating
+ * Abstraction over an RxJava {@link org.reactivestreams.Subscriber} that allows associating
```

Comment:
Actually it is not RxJava so `Abstraction over an {@link org.reactivestreams.Subscriber}`